app-misc/cal/cal-4.1.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="Display a monthly or yearly calendar, with optional appointments"
+DESCRIPTION="By default, cal  will display a calendar for the current month \
+with the current day marked. By specifing  certain arguments, cal will \
+display a calendar for a whole year or a specified month and year."
+HOMEPAGE=""
+COPYRIGHT="1993-2014, Alex Matulich, by Unicorn Research Corporation."
+LICENSE="BSD (4-clause)"
+REVISION="1"
+SOURCE_URI="http://unicorn.us.com/pub/cal41.zip"
+CHECKSUM_SHA256="96318b738970237832e4594cb363058f4243f54e86a5f1e06737fb5d02e6c298"
+SOURCE_DIR="cal41"
```

Comment:
We could drop `cd src` statements with this:

```diff
-SOURCE_DIR="cal41"
+SOURCE_DIR="cal41/src"
```

Also, the [Guidelines][0] forbids using `cd` to navigate to a package's main directory, but I'm not sure if it applies here.

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#source-directory

Reply:
Nice, didn't know this one yet :)

Reply:
It works :+1: